packages/shared/src/contexts/UITaskManager.tsx

```diff
@@ -4,23 +4,19 @@ import { EMPTY_LIST } from '@masknet/shared-base'
 import type { InjectedDialogProps } from './components'
 
 export interface ContextOptions<T, R> {
-    show(options: T, signal?: AbortSignal): Promise<R>
+    show(options?: Omit<T, 'open'>, signal?: AbortSignal): Promise<R>
 }
 
 export interface BaseDialogProps<T> extends Pick<InjectedDialogProps, 'open' | 'onClose'> {
-    onSubmit?(result: T): void
+    onSubmit?(result: T | null): void
 }
 
 /**
  * Create a manager of small UI task sessions,
  * which provide both a Context and a Provider.
```

Comment:
A grammar error written by me.
```suggestion
 * which provides both a Context and a Provider.
```